packages/demand-api/src/payment/types.ts

```diff
@@ -81,7 +84,7 @@ export type PaymentAuthResponse = {
   }
 }
 
-export type PaymentDetailsParams = {
+export type OldApiPaymentDetailsParams = {
```

Comment:
Let's not change a name - it is part of public API

Reply:
it hasn't been changed - its below